packages/design-system/src/components/Badge/primitive/BadgeFixed.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { forwardRef, Ref } from 'react';
+
+import BadgeBase, { BadgeProps, Label } from './BadgePrimitive';
+import { ButtonTertiary } from '../../Button';
+import Divider from '../../Divider';
+import Dropdown from '../../Dropdown';
+import { StackHorizontal } from '../../Stack';
+
+import style from './BadgePrimitive.module.scss';
+
+type BadgeWithDropdownProps = BadgeProps & {
+	values: string[];
+	defaultValue?: string;
+	onChange: (
+		event: React.MouseEvent<HTMLButtonElement> | KeyboardEvent,
+		value: { label: string },
+	) => void;
+};
+
+const Value = ({
```

Comment:
```suggestion
const BadgeValuePicker = ({
```

This should also be either in BadgePrimitive.tsx or in a dedicated BadgeValuePicker.tsx in `./Primitive`.  

Basically, we should only have a single primitive for the Badge itself. The different kind of badges are then `./variations`.